test/test-browser.js

```diff
@@ -9,8 +9,8 @@ const readFixtureDir = dir =>
 
 const content = `
 <script type="module">
-  import RemarkableWithMap from './dist/esm/index.js';
-  import RemarkableWithTextarea from './dist/esm/index.browser.js';
+  import { Remarkable as RemarkableWithMap } from './dist/esm/index.browser.js';
+  import { Remarkable as RemarkableWithTextarea } from './dist/esm/index.browser.js';
```

Comment:
I needed to fix the brower tests. I tested this locally. It should be fine now :+1: